egov/egov-egfweb/src/main/java/org/egov/egf/web/controller/microservice/VoucherController.java

```diff
@@ -47,6 +56,10 @@
 	private VoucherService voucherService;
 	@Autowired
 	private ChartOfAccountDetailService chartOfAccountDetailService;
+	
```

Comment:
Auto wiring is missing here. How the bean is getting injected? 
Use @Autowired annotation.